### ISSUE TITLE
Invalid integer: quipCount

### ISSUE THREAD

**Issue by MagnusdeAart** [tloncorp/landscape-apps#2641]
```
Error: Invalid integer: quipCount
    at Cu (https://magnus-nimpur.third.earth/apps/groups/assets/index.ad30a98e.js:1:24087)
    at ce (https://magnus-nimpur.third.earth/apps/groups/assets/index.ad30a98e.js:1:24488)
    at u (https://magnus-nimpur.third.earth/apps/groups/assets/index.ad30a98e.js:1:3435)
    at https://magnus-nimpur.third.earth/apps/groups/assets/index.ad30a98e.js:36:81758
    at Array.map (<anonymous>)
    at zh (https://magnus-nimpur.third.earth/apps/groups/assets/index.ad30a98e.js:36:81739)
    at $7 (https://magnus-nimpur.third.earth/apps/groups/assets/index.ad30a98e.js:48:229163)
    at Kl (https://magnus-nimpur.third.earth/apps/groups/assets/react-beautiful-dnd.fde4d763.js:19:52884)
    at mf (https://magnus-nimpur.third.earth/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:8723)
    at pf (https://magnus-nimpur.third.earth/apps/groups/assets/react-beautiful-dnd.fde4d763.js:23:947)
```

**Comment by jamesacklin** [comment]
Thanks. Can you paste the output of the output of `+vats %groups, =verb`? This should be fixed in the latest Groups update.

Otherwise, you can clear your IndexedDB cache by going into your browser's dev console and clearing the "key-value store" in "Application."

**Comment by jamesacklin** [comment]
Fixed by #2654
